src/odemis/odemisd/test/milling-series-test-missing-attributes.mill.yaml

```diff
@@ -0,0 +1,14 @@
+# Milling settings for testing purposes. Duration attribute is missing in this example
+"Missing milling duration": {
+    scanner: "FIB Scanner",
```

Comment:
This attribute doesn't exist.
Please make sure this YAML file is wrong for only 1 reason.  (the dicts are not inside a list, etc)